datanode/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Ivan Ermilov <ivan.s.ermilov@gmail.com>
 
 HEALTHCHECK CMD curl -f http://localhost:9864/ || exit 1
 
-ENV HDFS_CONF_dfs_datanode_data_dir=file:///hadoop/dfs/data
```

Comment:
@nevmmv why remove the file// prefix? did this cause any issues in version 3.2.1?

Reply:
@gmouchakis Yes. It did cause some error.

Reply:
@nevmmv do you remember the error? 
I've just tried it and it works with and without `file://`. Also in hadoop documentation (https://hadoop.apache.org/docs/r3.2.1/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml) the `file://` prefix is used in `dfs.namenode.name.dir` and `dfs.datanode.data.dir` so I'm thinking that maybe we should keep it this way.

Reply:
@gmouchakis This params was overrided in hadoop.env . Did you seen it?

Reply:
@nevmmv sorry, I haven't and I cannot find where it is overrided. hadoop.env does not contain the related entries and when I check the created containers it does not seem to get overrided.

Reply:
@gmouchakis I will check it and fix.

Reply:
@gmouchakis You were right, it works with prefix `file://`. I fixed this.